src/components/Resource/ResourceList.tsx

```diff
@@ -216,10 +219,10 @@ export default function ListView() {
           </div>
 
           <div className="mt-2 flex w-full flex-col gap-2 lg:w-fit lg:flex-row lg:gap-4">
-            <ButtonV2 className="py-[11px]" onClick={onBoardViewBtnClick}>
+            <Button variant={"primary"} onClick={onBoardViewBtnClick}>
               <CareIcon icon="l-list-ul" className="rotate-90" />
```

Comment:
add the margin right similar to what Shifting board has.

Reply:
Done, thanks for the heads-up, I missed it